ansys/mapdl/core/_commands/inq_func.py

```diff
@@ -0,0 +1,1281 @@
+"""Inquire undocumented functions"""
+
+class inq_function:
+
+    def ndinqr(self, node, key, pname='__tmpvar__', **kwargs):
+        """Get information about a node.
+
+        **Secondary Functions:**
+        Set current node pointer to this node.
+
+        .. warning:: **DISCLAIMER**: This function is un-documented in the official ANSYS Command Reference Guide.
+                   Hence its support is limited and it use is not encoraged.
+
+                   **Please use it with caution.**
+
+        Parameters
+        ----------
+        node  :  (int,  sc,  in)
```

Comment:
Please fix this here and for all other parameter descriptions matching this pattern. Should be a simple find and replace.
```suggestion
        node : int or str
```

Reply:
Done.